apps/utils/locust_tests/common/db.py

```diff
@@ -32,6 +39,44 @@ def _execute(uri: str, query: str) -> List:
     return results
 
 
+def __get_regression_query(select_query: str) -> str:
+    return " ".join(
+        [
+            select_query,
+            f'WHERE "bene_id" BETWEEN {REGRESSION_BENE_RANGE_START} AND {REGRESSION_BENE_RANGE_END}',
+            'ORDER BY "bene_id" ASC',
+            f"LIMIT {LIMIT}",
+        ]
+    )
+
+
+def get_regression_bene_ids(uri: str) -> List[str]:
+    bene_query = __get_regression_query('SELECT "bene_id" FROM "beneficiaries"')
+    return [str(r[0]) for r in _execute(uri, bene_query)]
+
+
+def get_regression_hashed_mbis(uri: str) -> List[str]:
+    mbi_query = __get_regression_query('SELECT "mbi_hash" FROM "beneficiaries"')
+    return [str(r[0]) for r in _execute(uri, mbi_query)]
```

Comment:
These functions are used by each regression suite to load data -- each of them calls `__get_regression_query()` with a `SELECT` query string as the value for `select_query`:

```python
def _get_regression_query(select_query: str) -> str:
    return " ".join(
        [
            select_query,
            f'WHERE "bene_id" BETWEEN {REGRESSION_BENE_RANGE_START} AND {REGRESSION_BENE_RANGE_END}',
            'ORDER BY "bene_id" ASC',
            f"LIMIT {LIMIT}",
        ]
    )
```

`__get_regression_query()` is fairly simple, it just concatenates the `SELECT` query with: 

- A `WHERE` clause that limits the query to the range of 19,999 synthetic beneficiaries that exist in each environment
- An `ORDER BY` clause that orders the results by beneficiary ID in ascending order (so that the data is always consistently ordered)
- And a `LIMIT` clause that puts a hard 100,000 result limit on the returned results

So for the two functions here, `get_regression_bene_ids()` and `get_regression_hashed_mbis()`, both are already limited by the range of 19,999 synthetic beneficiaries and by the global limit of 100,000 results (via the `LIMIT` constant in `db.py`).

Does that answer your question? Or, are you talking about a different kind of limit here?

Reply:
That answers it, it's basically implicitly limited by the amount of data. I think its fine as-is then, given i doubt we want to adjust the data we're using for now.